plugins/matomo-backend/src/service/router.ts

```diff
@@ -0,0 +1,60 @@
+import { errorHandler } from '@backstage/backend-common';
+import { Config } from '@backstage/config';
+import express from 'express';
+import Router from 'express-promise-router';
+import { createProxyMiddleware } from 'http-proxy-middleware';
+import { Logger } from 'winston';
+
+export interface RouterOptions {
+  logger: Logger;
+  config: Config;
+}
+
+export async function createRouter(
+  options: RouterOptions,
+): Promise<express.Router> {
+  const { config } = options;
+
+  const matomoToken = config.getString('matomo.api_token');
+  const matomoApiUrl = config.getString('matomo.api_url');
```

Comment:
This review still applies. Please add this to the config schema.

https://github.com/janus-idp/backstage-plugins/pull/370#discussion_r1197909311